sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -204,11 +246,27 @@ upstream source must provide valid data on every cycle that ready is asserted.
 This is equivalent to changing the pipe read calls to a non-blocking call and assuming that
 success is always true.
 
-If set to `false`, the `min_capacity` pipe class template parameter and `ready_latency`
-property must be 0.
+This property is not guaranteed to be respected if the pipe is an inter-kernel
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*
+or *avalon_mm*.
+
+|`uses_ready`
```

Comment:
We should make it clear that this property is only applicable if the outside world would be the one driving the signal (i.e. on kernel-to-host/IO pipes).